packages/@aws-cdk/aws-rds/lib/parameter-group.ts

```diff
@@ -21,7 +29,7 @@ abstract class ParameterGroupBase extends Resource implements IParameterGroup {
    * Imports a parameter group
    */
   public static fromParameterGroupName(scope: Construct, id: string, parameterGroupName: string): IParameterGroup {
-    class Import extends Resource implements IParameterGroup {
+    class Import extends ParameterGroupBase implements IParameterGroup {
```

Comment:
Why modify this? Imported parameter groups are ones that are defined elsewhere but are imported into the CDK as read-only. It's not possible to modify an imported parameter group.

Reply:
I didn't want to define an empty `addParameter` method there and since that's now part of `IParameterGroup` it's required. Do you have any idea to solve this neatly or should I just opt for that empty method?

Reply:
Ah, I see what happened. 

The better to way to fix this is -
1. drop the `addParameter` method on the `IParameterGroup` interface.
2. change the code at line 439 of `cluster.ts` to 
   ```ts
    if (!clusterParameterGroup) {
      clusterParameterGroup = new ClusterParameterGroup(this, "ClusterParameterGroup", {
        family: props.engine.getClusterParameterGroupFamily(props.engineVersion)
      });
    } else if (clusterParameterGroup instanceof ClusterParameterGroup) { // Skip modification on an imported parameter group
      if (s3ImportRole) {
        clusterParameterGroup.addParameter('aurora_load_from_s3_role', s3ImportRole.roleArn);
      }
      if (s3ExportRole) {
        clusterParameterGroup.addParameter('aurora_select_into_s3_role', s3ExportRole.roleArn);
      }
    }
   ```

Does that make sense?